app/routes/table.$version._index.tsx

```diff
@@ -44,7 +46,7 @@ const menu = [
         <FaBook className="text-lg" /> Docs
       </div>
     ),
-    to: './docs/guide/introduction',
+    to: './docs/introduction',
```

Comment:
Are we moving this doc to a new place in the `tanstack/table` repo too? Right now we only have `./docs/guide/introduction`: https://github.com/TanStack/table/blob/main/docs/guide/introduction.md 

`/table/v8/docs/introduction` shows a `File does not exist` error message for me:
<img width="1041" alt="image" src="https://github.com/TanStack/tanstack.com/assets/43729152/29106634-7738-4028-b994-ff1357cd1be7">

Reply:
Yes sorry it's this PR here: https://github.com/TanStack/table/pull/5295

Reply:
Ahhh, got it!